media/views/common/suggestionHeader.scss

```diff
@@ -4,6 +4,10 @@
   flex-direction: column;
   width: 100%;
   height: 100%;
+
+  &--header {
+    padding-bottom: 6px;
```

Comment:
Is there a need in that? We have padding 20px applied for all sections. This change makes it inconsistent 🤔 Maybe having half of 20px would look better, rather than 20px vs. 6px difference.

Reply:
Happy to put this to 10px
**Some detail as to why I did this:**
The core issue with spacing here is caused by SnykLearn being asynchronous and not guaranteed to be present.
I put in a fixed height to prevent jerky content movement caused by the SnykLearn button being rendered. But if SnykLearn CTA never renders, there is an awkwardly large gap. The cleanest it looked was with less padding.

Reply:
Discussed with Michel, Will change it to expand height and remove inconsistent padding.